docs/distributions/manjaro/faq.md

```diff
@@ -1,30 +1,60 @@
+# Migrating to new kernel
+
+Run this in your terminal:
+
+```sh
+cat << EOF | sudo tee -a /etc/pacman.conf
+[manjaro-mact2]
+SigLevel = Never
+Server = https://mirror.funami.tech/$repo/os/$arch
+EOF
+sudo pacman -Syu linux516-t2 linux516-t2-headers dkms apple-bce-dkms-git apple-ibridge-dkms-git apple-bcm-wifi-firmware
```

Comment:
Does this need `--ignore zfs-utils` to get temporarily around the dependency issues

Reply:
i dont think so but for safety i'll add